src/commander/commander.c

```diff
@@ -318,6 +340,21 @@ static commander_error_t _api_process(const Request* request, Response* response
     case Request_restore_from_mnemonic_tag:
         response->which_response = Response_success_tag;
         return _api_restore_from_mnemonic(&(request->request.restore_from_mnemonic));
+#endif
+#if defined(FIRMWARE_BASE)
+    case Request_display_base32_tag:
```

Comment:
same as my other  comments: here you can collapse all into one case: `Request_bitboxbase_tag`

Reply:
ack